doc/install.md

```diff
@@ -0,0 +1,145 @@
+[doc](../README.md) > Installation
+
+This system consists of two elements: station (which is intended to run on a Raspberry Pi with an SDR dongle, but can
+be run on any Linux box) and a server (which is intended to be run in a place with good uplink connectivity). If 
+you are interested in running your own station, you most likely want to deploy just the station and use existing
+server. Please contact someone from the satnogs team and we'll hook you up.
+
+# Station installation
+
+You need to install [Rasbian](https://www.raspbian.org/) on your Raspberry Pi. Please follow any Raspbian installation
+instruction, such as [this one](https://www.raspberrypi.org/documentation/installation/installing-images/). Once
+done, connect to your Pi and do the following as root:
+
+1. **Install necessary dependencies**:
+
+```
+apt update
+apt install python3-minimal git rtl-sdr sox imagemagick
+```
+
+Also, the following tools are needed:
+
+1. noaa-apt - download Raspberry Pi binaries from https://noaa-apt.mbernardi.com.ar/download.html
+2. medet - https://github.com/artlav/meteor_decoder, binaries available from: http://orbides.org/page.php?id=1023
+3. meteor-demod - https://github.com/artlav/meteor_decoder
+
+2. **Create satnogs user**:
+
+```
+# useradd satnogs
+```
+
+3. Now **switch to the satnogs user**:
+```
+su - satnogs
+```
+
+4. **Get the latest satnogs-gut code**. This and following steps should be done as satnogs user.
+
+```
+git clone https://github.com/gut-space/satnogs
+```
+
+5. **Install python dependencies**:
+
+```
+cd satnogs/station
+python3 setup.py install
+```
+
+This step will install necessary dependencies. It is a good practice to install them in virtual environment. However,
+since the scripts will be called using crontab, it would've complicated the setup.
+
+5. **Run the initial setup** script:
+
+```
+python3 station/setup.py
```

Comment:
User call this script in previous step. Anyway call setup.py without arguments do nothing.